cursor_test.go

```diff
@@ -0,0 +1,100 @@
+package apicursor
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"testing"
+	"time"
+)
+
+func TestAddCursorFiltersMulti(t *testing.T) {
```

Comment:
We should probably update the README with the new syntax method for UnmarshalMongo